documents/invoice/bootstrapper.go

```diff
@@ -55,7 +55,13 @@ func (Bootstrapper) Bootstrap(ctx map[string]interface{}) error {
 	srv := DefaultService(
 		docSrv,
 		repo,
-		queueSrv, txManager)
+		queueSrv, txManager, func() documents.TokenRegistry {
+			tokenRegistry, ok := ctx[bootstrap.BootstrappedInvoiceUnpaid].(documents.TokenRegistry)
+			if !ok {
+				panic("token registry initialisation error")
```

Comment:
Don't we usually return error instead? it is easier to build tests for bootstrappers.

Reply:
that would be problematic, since we must not run the node in a misconfigured state. For config errors we always panic.

Reply:
Isn't it the duty of the main boostrapper? to panic when errors while iterating over all the individual bootstrappers?

Reply:
this is a special case since the token registry needs to be injected at a later point because nft package is bootstrapped after the invoice or po packages, hence the closure.

Reply:
ah didnt catch that, thanks.